src/adapters/git.ts

```diff
@@ -61,7 +61,7 @@ abstract class GitAdapter implements IRepoAdapter {
   public async commitRepo(repo: IRepo): Promise<void> {
     const { migration: { spec } } = this.migrationContext;
     await this.git(repo).add('.');
-    await this.git(repo).commit(`[shepherd] ${spec.title}`);
+    await this.git(repo).commit(`${spec.title} [shepherd]`);
```

Comment:
Any opposition to removing the branding entirely?

```suggestion
    await this.git(repo).commit(spec.title);
```

At NerdWallet we still like to call out shepherd migrations in the title, but we can prefix `[shepherd]` manually in our spec titles.

Reply:
I think we've got consensus, let's do it!